docs/modules/develop/pages/backports.adoc

```diff
@@ -0,0 +1,14 @@
+= Backports
+
+We only offer support for the last two mayor version.
+
+For instance, if we're on the version `0.28.0.dev`, we make backports for `v0.26.x` and `v0.27.x`. We only backport fixes and not features, as we want to respect xref:develop:guide_conventions.adoc[Semantic Versioning].
+
+The idea of a backport is to bring a given fix from the last development version to an older release.
+
+The process for making backports is the following:
+
+. Check in the git commit history what was the latest backports on the branch that you're working with. https://github.com/decidim/decidim/commits/release/0.27-stable[Example for v0.27]. Copy the number of the last PR backported.
+. Search inn GitHub for the this PR in the list of merged PRs of the `type: fix`. https://github.com/decidim/decidim/pulls?page=1&q=is%3Apr+sort%3Aupdated-desc+label%3A%22type%3A+fix%22+is%3Amerged[Example URL].
+. Check the PRs merged just before and after this one. You need to find the last ones that weren't backported already to start with the backport process. A fast way for checking this out is to open the PRs and see the mentions from the backports in the GitHub references.
+. Once you have the list of the PRs that you want to backport, you can start with the process by using the https://github.com/decidim/decidim/blob/develop/bin/backporter[backporter script]. This script mostly handles the branch creation, cherrypicking, labeling and pushing of the fix. For using it you'll need to have a GitHub Personal Access Token.
```

Comment:
I think that you could add some command examples : 

```
# example
backporter --github-token=GITHUB_TOKEN --pull-request-id=10248 --version-number=0.27
```

Reply:
Added in 9dd9ea6b6f25815e5a51228c4d046fae4c4cf794 with the documentation